drivers/hud_spm_report/app/models/hud_spm_report/generators/fy2024/hdx_upload.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
+
```

Comment:
This needs to go below the copyright or our update script will generate duplicate copyright notices.

Reply:
thanks, I feel like we should probably fix our script right? Seems like it should handle this kind of thing. I'll make an issue to adjust it